frontend/pages/register/index.tsx

```diff
@@ -0,0 +1,7 @@
+export default function Register() {
```

Comment:
Was this potentially added back in re: merge conflicts? This file shouldn't exist anymore - it was replaced with `viewer-registration`